portfolio/src/main/java/com/google/sps/servlets/UserInformationServlet.java

```diff
@@ -0,0 +1,141 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Uses UserService and DataStore to allow Creation, Reading, and Updating of User objects
+ * JSON is sent on GET, on POST create a new user object or update individual parameters
+ */
```

Comment:
As mentioned in your code comment, as by design, you have expectation that userInfo can be retrieved by anyone.

What do you think about making this point as a class content, that user data is accessible by anyone by design, and it should not contain any private information.